### ISSUE TITLE
Trumbowyg not loading assets

### ISSUE THREAD

**Issue by arinhouck** [wearefine/fae#290]
I tried importing: 
```
//= require fae/vendor/trumbowyg
```

Error:
```
couldn't find file 'fae/vendor/trumbowyg' with type 'application/javascript'
Checked in these paths: 
  /Users/Shared/rails_projects/portfolio/app/assets/config
  /Users/Shared/rails_projects/portfolio/app/assets/images
  /Users/Shared/rails_projects/portfolio/app/assets/javascripts
  /Users/Shared/rails_projects/portfolio/app/assets/stylesheets
```

So I included trumbowyg manually as a temporary solution. However it appears with the option `html: true` is not rendering the html editor. Still a regular input. Any thoughts?

Ruby 2.3.1
Rails 5.0.0
Fae Rails 1.5.1
Development Env

**Comment by aaronransley** [comment]
Did you add the `//= require fae/vendor/trumbowyg` line to the `app/assets/javascripts/fae.js` file in your app? I believe that the `fae.js` file is uniquely bound to the Fae admin, and can access the vendor assets within the Fae gem during asset compilation.

**Comment by arinhouck** [comment]
Yes the line above was imported on `fae.js`. @aaronransley and the above error was presented after importing.

**Comment by aaronransley** [comment]
Hmm, that's about the extent of my knowledge when it comes to the Trumbowyg integration. Sorry, Arin!

@6temes or @jamesmk may have more ideas, however?

**Comment by 6temes** [comment]
I created a new app to reproduce the error and I found out that the version from Rubygems is somehow broken, but the current master works well.

https://github.com/6temes/fae_error

So, for the moment, the workaround would be to use:

`gem 'fae-rails', git: 'https://github.com/wearefine/fae.git'`

**Comment by arinhouck** [comment]
Thanks @aaronransley @6temes. I will be on the lookout for the next release. This was a great alternative to using refinery-cms for my personal portfolio. Just went live this evening with fae.

**Comment by aaronransley** [comment]
Thanks for jumping in, @6temes. I'll make a note to look into Fae's latest gem release having broken Trumbowyg support. 

@arinhouck congrats on the launch and feel free to share a link if you'd like. Always fun to see what folks are building with Fae :)

**Comment by arinhouck** [comment]
@aaronransley Much appreciated. The new setup is live on http://www.arinhouck.com/

**Comment by jamesmk** [comment]
Thanks for the report @arinhouck and congrats on the launch! This feature is currently unreleased and will be included in v1.6 coming out in a couple weeks. For now you can follow @6temes' advice and install the gem directly from the repo to use it.

Hope this helps!

thanks
